src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java

```diff
@@ -1211,6 +1214,14 @@ public void testFourPartSyntaxCallEscapeSyntax() throws SQLException {
             cstmt.execute();
             assertEquals(sum, cstmt.getInt(3));
         }
+
+        try (CallableStatement cstmt = connection.prepareCall("{? = call [" + linkedServer + "].master.dbo.TestReturn(?)}")) {
+            int expected = 15;
```

Comment:
No need for a new variable, its only used in one place.

Reply:
Defined it as "expected" to give meaning